src/tree/ResolvableTreeItemBase.ts

```diff
@@ -46,13 +48,14 @@ export abstract class ResolvableTreeItemBase extends AzExtParentTreeItem impleme
         return false;
     }
 
-    public async resolve(clearCache: boolean, context: IActionContext): Promise<void> {
+    public async resolve(clearCache: boolean, _context: IActionContext): Promise<void> {
         if (!this.resolveResult || clearCache) {
             ext.activationManager.onNodeTypeResolved(this.data.type);
 
             const resolver = this.getResolver();
 
-            await this.runWithTemporaryDescription(context, 'Loading...', async () => {
```

Comment:
A couple of thoughts...

1. Should we change `runWithTemporaryDescription` to also refresh the parent instead?
    1. Relatedly...if VSCode fixes https://github.com/microsoft/vscode/issues/121567, what happens if we do that? 
1. As an alternative, we could provide an option to `runWithTemporaryDescription` that eliminates the spinner icon, instead of implementing a sorta-custom version here.